applications/nublado/values.yaml

```diff
@@ -244,100 +244,165 @@ hub:
 # Configuration for the Zero to JupyterHub subchart.
 jupyterhub:
   hub:
+    # -- Whether to require metrics requests to be authenticated
     authenticatePrometheus: false
+
     image:
+      # -- Image to use for JupyterHub
       name: ghcr.io/lsst-sqre/rsp-restspawner
+
+      # -- Tag of image to use for JupyterHub
       tag: 0.2.0
+
+    # -- Resource limits and requests
     resources:
       limits:
         cpu: 900m
         memory: 1Gi  # Should support about 200 users
+
     db:
-      # Password comes from the nublado-secret.
+      # -- Type of database to use
       type: "postgres"
+
+      # -- Database password (not used)
+      # @default -- Comes from nublado-secret
       password: "true"
+
+      # -- URL of PostgreSQL server
+      # @default -- Use the in-cluster PostgreSQL installed by Phalanx
       url: "postgresql://jovyan@postgres.postgres/jupyterhub"
+
+    # -- Security context for JupyterHub container
     containerSecurityContext:
       runAsUser: 768
       runAsGroup: 768
       allowPrivilegeEscalation: false
+
+    # -- Base URL on which JupyterHub listens
     baseUrl: "/nb"
+
+    # -- Existing secret to use for private keys
     existingSecret: "nublado-secret"
-    # This will need work
+
+    # -- Additional environment variables to set
+    # @default -- Gets `JUPYTERHUB_CRYPT_KEY` from `nublado-secret`
     extraEnv:
       JUPYTERHUB_CRYPT_KEY:
         valueFrom:
           secretKeyRef:
             name: "nublado-secret"
             key: "hub.config.CryptKeeper.keys"
+
+    # -- Additional volumes to make available to JupyterHub
+    # @default -- The `hub-config` `ConfigMap` and the Gafaelfawr token
     extraVolumes:
       - name: "hub-config"
         configMap:
           name: "hub-config"
       - name: "hub-gafaelfawr-token"
         secret:
           secretName: "hub-gafaelfawr-token"
+
+    # -- Additional volume mounts for JupyterHub
+    # @default -- `hub-config` and the Gafaelfawr token
     extraVolumeMounts:
       - name: "hub-config"
         mountPath: "/usr/local/etc/jupyterhub/jupyterhub_config.d"
       - name: "hub-gafaelfawr-token"
         mountPath: "/etc/gafaelfawr"
-    # We still have to use our own, enabled at the top level, which is
-    #  similar but not identical.  This one still doesn't work, even if
-    #  you explicitly enable port 8081 so the labs can talk to the Hub.
+
     networkPolicy:
+      # -- Whether to enable the default `NetworkPolicy` (currently, the
+      # upstream one does not work correctly)
       enabled: false
+
     loadRoles:
-      self:
-        scopes: ['admin:servers!user', 'read:metrics']
       server:
-        scopes: ['inherit']  # Let server use API like user
+        # -- Default scopes for the user's lab, overridden to allow the lab to
+        # delete itself (which we use for our added menu items)
+        scopes: ["self"]
```

Comment:
I think if you do this it is going to break the items we added for save?-and-exit-and-logout?